detect_secrets/core/usage/audit.py

```diff
@@ -46,6 +47,34 @@ def _add_mode_parser(parser: argparse.ArgumentParser) -> None:
     )
 
 
+def _add_report_parser(parser: argparse.ArgumentParser) -> None:
+    parser.add_argument(
+        '--report',
+        action='store_true',
+        help=(
+            'Displays a report with the secrets detected'
+        )        
+    )
+
+    parser.add_argument(
+        '--only-real',
+        action='store_true',
+        help=(
+            'Only includes real secrets in the report'
+        )
+    )
+
+    parser.add_argument(
+        '--only-false',
+        action='store_true',
+        help=(
+            'Only includes false positives in the report'
+        )
+    )
```

Comment:
Might be worth using https://docs.python.org/3/library/argparse.html#mutual-exclusion, since it doesn't really make sense for `--only-real --only-false` to be used together.

Reply:
Done!